glean/load.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// clang-format off
+// This file exposes a list of metrics/pings to the QML/JS world.
+.pragma library
+// Importing Glean.js as released by the Glean team.
+.import 'glean.js' as RealGlean
+.import 'generated/pings.js' as Pings
+.import 'generated/sample.js' as Sample
+// clang-format on
+
+// public functions
+
+// In this way we expose the samples to what imports the module.
+const sample = Sample;
+
+function initialize(appVersion, enabled, uploader) {
+  RealGlean.Glean.default.initialize('MozillaVPN', enabled, {
+    appBuild: `MozillaVPN/${appVersion}`,
+    httpClient: uploader,
+    // TODO: this should be removed
+    debug: {logPings: true}
+  });
+}
+
+function sendPing() {
+  Pings.ping.submit();
+}
+
+function setUploadEnabled(enabled) {
```

Comment:
Is this ever called? Can users toggle telemetry collection on/off during the application lifecycle?